### DISCUSSION TITLE
Welcome to ctlos Discussions!

### DISCUSSION THREAD

**Discussion by creio** [ctlos/.github#2] [Общее]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by wzooroo** [comment]
Скажите, а nvidia 340 драйвера при установки можно как то добавить в образ? Прям обидно как то за линукс((( Не на винду же переходить(

**Reply by creio** [reply]
Все это можно установить после, просто эти пакеты в aur.

**Comment by Ndashka-Dev** [comment]
Есть ли в планах в меню оконных менеджеров во время установки добавить пункты Qtile и/или Wayfire?

**Comment by als-creator** [comment]
кроме sourceforge где то есть iso с релизами? sourceforge заблочили, хотел поставить на пробу
